# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/alanw/chrony/chrony_devfs.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur chrony-1.19.orig/conf.c chrony-1.19/conf.c
--- chrony-1.19.orig/conf.c	2002-03-01 00:27:09.000000000 +0100
+++ chrony-1.19/conf.c	2003-03-07 20:21:56.000000000 +0100
@@ -85,7 +85,7 @@
 /* ================================================== */
 /* Configuration variables */
 
-static char *rtc_device = "/dev/rtc";
+static char *rtc_device = "/dev/misc/rtc";
 static int ntp_port = 123;
 static char *keys_file = NULL;
 static char *drift_file = NULL;
